.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Update IntelliJ Version
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
```

Comment:
Pls document the meaning of the cron in human-readable form, as in https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/.circleci/config.yml#L302